### DISCUSSION TITLE
Water mist and prescribed fire

### DISCUSSION THREAD

**Discussion by FireLotus17** [firemodels/fds#13452] [General]
Hi all,
I've been monitoring discussions on the topic of water mist for a while now. As we know, the extinguishing mechanism of water mist is typically based on heat absorption due to its latent heat of vaporization. Subsequently, there is the possibility of oxygen reduction, and finally, there is a negligible chance of wetting. However, when I prescribe an HRR curve for a fire in the presence of water mist, unless oxygen reduction phenomena occur (as in the validation guide, case HAI, for example), the HRR curve does not tend to decrease, even though FDS, through QPART, is able to estimate the heat absorbed by the water mist particles. Surely, I must be missing something, but if QPART is equivalent (in the negative) to the HRR, shouldn't I see an absence of flames? 
Then there is another issue, perhaps more philosophical in nature, that I wonder about. When we talk about HRR, we are used to referring to the source (usually characterized through oxygen reduction calorimetry), but in the presence of active protection, wouldn't it be appropriate to think in terms of heat due to HRR+QPART?
Thanks

**Comment by drjfloyd** [comment]
The primary mechanism is reduction of oxygen. Look at the coast guard water mist cases. In open door tests, the fire does not go out even when the water mist is capable of maintaining relatively low temperatures.

**Comment by FireLotus17** [comment]
Ok. Thank you very much for the response. But in some way, shouldn't the power absorbed due to the latent heat of vaporization promote surface cooling and thus extinguishment? I'm asking these questions because I've been seeing many publications where the e_coefficient is included, which I haven't found in the Coast Guard cases. However, I'm quite doubtful about the use of the e_coefficient, as it's essentially like computing a double power drop: one induced by the e_coefficient and the other by Q_PART. What do you think?

**Comment by drjfloyd** [comment]
Watermist systems provide cooling of the flame, prewetting of non-burning surfaces, absorption of thermal radiation, and dilution of oxygen; however, the last effect is generally the dominant effect:

" Braidech [l] concluded correctly in 1955 that the suppression effect of water mist "appears to be due pre-dominantly to dilution of the air (oxygen) supply in the zone of burning with vapour (steam). . ." That is to say, oxygen displacement appears to play a stronger role than flame cooling." [1]

[1] https://www.researchgate.net/publication/44049807_A_Closer_Look_At_The_Fire_Extinguishing_Properties_Of_Water_Mist
